databases/week2/Part2_School_Database.sql

```diff
@@ -0,0 +1,31 @@
+/* Create a new database containing the following tables:
+    Class: with the columns: id, name, begins (date), ends (date)
+    Student: with the columns: id, name, email, phone, class_id (foreign key) */ 
+
+
+CREATE DATABASE school_database;
+Use school_database;
+CREATE TABLE class (
+  id int(10) unsigned NOT NULL AUTO_INCREMENT,
+  name varchar(255) NOT NULL,
+  begins DATETIME NOT NULL,
+  ends DATETIME NOT NULL,
+  PRIMARY KEY (id)
+);
+CREATE TABLE student (
+  id int(10) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
This works, just keep in mind https://stackoverflow.com/questions/58938358/mysql-warning-1681-integer-display-width-is-deprecated